video_creation/final_video.py

```diff
@@ -34,7 +35,15 @@ def name_normalize(name: str) -> str:
     name = re.sub(r"([0-9]+)\s?\/\s?([0-9]+)", r"\1 of \2", name)
     name = re.sub(r"(\w+)\s?\/\s?(\w+)", r"\1 or \2", name)
     name = re.sub(r"\/", r"", name)
-    return name
+    
+    lang = settings.config["reddit"]["thread"]["post_lang"]
+    if lang != "":
```

Comment:
This can be shortened, and it will also handle None values (which should never happen, but regardless)
```suggestion
    if lang:
```